octobot_trading/modes/script_keywords/dsl/__init__.py

```diff
@@ -0,0 +1,27 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
```

Comment:
What `dsl` mean?

Reply:
domain specific language, because there we are parsing things (like `6%a`, this kind of thing). you think we should name it differently ?

Reply:
Ok, the full name would be too long, we can keep `dsl` :)